### DISCUSSION TITLE
My node is installed but in spatie it is not detected

### DISCUSSION THREAD

**Discussion by ffadelm** [spatie/laravel-pdf#105] [Q&A]
The command "node "D:\programming\laragon\www\invoiceradian\vendor\spatie\browsershot\src/../bin/browser.cjs" "{""url"":""file:\/\/C:\\Users\\fadel\\AppData\\Local\\Temp\\1530619931-0997831001710212810\\index.html"",""action"":""pdf"",""options"":{""args"":[],""viewport"":{""width"":800,""height"":600},""displayHeaderFooter"":false,""printBackground"":true}}"" failed. Exit Code: 1(General error) Working directory: D:\programming\laragon\www\invoiceradian\public Output: ================ Error Output: ================ 'node' is not recognized as an internal or external command, operable program or batch file.

![image](https://github.com/spatie/laravel-pdf/assets/67732382/7f204f1b-bb7c-4ac3-921c-17f8c777a982)

**Comment by Professor93M** [comment]
same with me
